executor/load_data.go

```diff
@@ -473,7 +474,7 @@ func (e *LoadDataInfo) colsToRow(ctx context.Context, cols []field) []types.Datu
 		if cols[i].maybeNull && string(cols[i].str) == "N" {
 			e.row[i].SetNull()
 		} else {
-			e.row[i].SetString(string(cols[i].str))
+			e.row[i].SetString(string(cols[i].str), collate.DefaultCollation, collate.DefaultLen)
```

Comment:
Why use default？

Reply:
Here, haven't touch the column information. So no column filed type information can be got.